build/Build.cs

```diff
@@ -99,8 +100,15 @@ class Build : NukeBuild
         .DependsOn(Compile)
         .Executes(() =>
         {
-            var assembly = RootDirectory / "bin" / Configuration / $"{ProjectName}.dll";
-            CopyFileToDirectory(assembly, DnnBinDirectory, policy: FileExistsPolicy.Overwrite);
+            var files = RootDirectory.GlobFiles(
+                $"bin/Debug/*{ProjectName}*.dll",
```

Comment:
Here we should use {Configuration} instead of debug or it will chocke when built in release mode in CI

Reply:
This is on `Deploy` target.